packages/core/test/util/transactionSummaryInspector.test.ts

```diff
@@ -271,8 +271,15 @@ describe('Transaction Summary Inspector', () => {
             ]),
             coins: 6_000_000n
           }
+        },
+        {
+          address: addresses[0],
+          value: {
+            coins: 3_000_000n
+          }
```

Comment:
It would be nice to demonstrate how a withdrawal can change the amount of `coins` in the inspection
```suggestion
          address: addresses[0],
          value: {
            coins: 2_000_000n
          }
        },
        {
          address: externalAddress1,
          value: { coins: 1_000_000n }
```

Reply:
Updated here: [3897f94](https://github.com/input-output-hk/cardano-js-sdk/pull/1548/commits/3897f945eea3f4da83fb9e9d52aa99884cb54338)